lib/remote/configstageshandler.cpp

```diff
@@ -13,6 +13,8 @@ using namespace icinga;
 REGISTER_URLHANDLER("/v1/config/stages", ConfigStagesHandler);
 
 std::atomic<bool> ConfigStagesHandler::m_RunningPackageUpdates (false);
+// A timestamp that indicates the last time an Icinga 2 reload failed.
+static Atomic<double> l_LastReloadFailedTime(0);
```

Comment:
Why is one a static class member and the other a static compilation unit variable? They belong together and are used in the same places, so this should be consistent.